src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -668,7 +668,7 @@ public function findHostsByIdsForAdminUser(array $hostIds): array
                 $hosts[] = $host;
             }
         } else {
-            throw new \Exception(_('Bad SQL request'));
+            throw new \Exception(_('Failed to find hosts'));
```

Comment:
No need to throw an exception. If an exception is thrown by PDO it is up to the service to handle the case.

Reply:
done :heavy_check_mark: